src/components/share-overlay/share-overlay.js

```diff
@@ -1,20 +1,20 @@
 //@flow
-import {h} from 'preact';
+import {h, Component} from 'preact';
 import {Text, Localizer} from 'preact-i18n';
 import {connect} from 'preact-redux';
 import {bindActions} from '../../utils/bind-actions';
 import {actions} from '../../reducers/share';
 import {toHHMMSS, toSecondsFromHHMMSS} from '../../utils/time-format';
-import BaseComponent from '../base';
 import {Overlay} from '../overlay';
 import {default as Icon, IconType} from '../icon';
 import style from '../../styles/style.scss';
 import {CopyButton} from '../copy-button/copy-button';
+import {withLogger} from 'components/logger';
```

Comment:
import {withPlayer} from '../player';

Reply:
share overlay get its player as prop already from before, from the share component